### PR TITLE
docs: add initial template for PR's

### PR DESCRIPTION
**Pull request by mhamwala**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+For more information on contributing to the Appsody Docs, see the contributors guide: 
+https://github.com/docs/CONTRIBUTING.md
+-->
+
+##### Checklist
+<!-- For completed items, change [ ] to [x]. -->
+
+- [ ] commit message follows [commit guidelines](https://github.com/appsody/docs/blob/master/CONTRIBUTING.md#commit-message-guidelines)
+
```
Comment (helenmasters) [inline .github/PULL_REQUEST_TEMPLATE.md]: You might want to consider whether it's useful to add the following to your checklist:

If applicable, I have updated the documentation accordingly.
If applicable, I have added tests to cover my changes.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!--
+Thank you for your pull request. Please provide a description above and review
+the requirements below.
+
+For more information on contributing to the Appsody Docs, see the contributors guide:
```
Comment (helenmasters) [inline .github/PULL_REQUEST_TEMPLATE.md]: either - contributor's guide or contributing guidelines

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mhamwala) [reply .github/PULL_REQUEST_TEMPLATE.md]: yep, I was trying to keep this checklist specific to the docs repo, I will carry these suggestions when creating the other templates

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (mhamwala) [reply .github/PULL_REQUEST_TEMPLATE.md]: change addressed

**Pull request merged**